gradlew

```diff
@@ -125,10 +126,11 @@ if $darwin; then
     GRADLE_OPTS="$GRADLE_OPTS \"-Xdock:name=$APP_NAME\" \"-Xdock:icon=$APP_HOME/media/gradle.icns\""
 fi
 
-# For Cygwin, switch paths to Windows format before running java
-if $cygwin ; then
+# For Cygwin or MSYS, switch paths to Windows format before running java
+if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
```

Comment:
Did you upgrade gradlew to be compatible with maven?

Reply:
No, I upgraded the Gradle versions to be able to use the "publish-plugin" and current version of the "java" plugin.

- https://github.com/ePages-de/restdocs-api-spec/blob/18d8af911eec7e24a07aeef50e3079735ae9909c/build.gradle.kts#L10
- https://github.com/ePages-de/restdocs-api-spec/blob/18d8af911eec7e24a07aeef50e3079735ae9909c/restdocs-api-spec/build.gradle.kts#L74-L77

Reply:
And using a recent version is always good ;)